lib/db/models/Referral.ts

```diff
@@ -1,13 +1,21 @@
 import { DataTypes, Model, Sequelize } from 'sequelize';
-import { SwapType } from '../../consts/Enums';
+import { OrderSide, SwapType } from '../../consts/Enums';
 
 type Limits = {
   minimal?: number;
   maximal?: number;
 };
 
-// TODO: direction of chain swaps
-type Premiums = Partial<Record<SwapType, number>>;
+type DirectionalPremium = {
+  [OrderSide.BUY]: number;
+  [OrderSide.SELL]: number;
+};
```

Comment:
```suggestion
type DirectionalPremium = Partial<{
  [OrderSide.BUY]: number;
  [OrderSide.SELL]: number;
}>;
```

To allow setting only one side

Reply:
I'd prefer if premiums per swap type were set together as it's easier on the client